aiqc/__init__.py

```diff
@@ -1018,14 +1040,18 @@ def to_numpy(
 			samples:list = None
 		):
 			df = Dataset.Tabular.to_pandas(id, columns, samples)
+
+			if Dataset.Text.column_name not in columns:
+				return df.to_numpy()
+
 			word_counts, feature_names = Dataset.Text.get_feature_matrix(df)
-			return word_counts, feature_names
+			return word_counts.todense()
 
 
 		def get_feature_matrix(
 			dataframe:object
 		):
-			count_vect = CountVectorizer()
+			count_vect = CountVectorizer(max_features = 200)
```

Comment:
Maybe we could expose a `**kwargs` for situations like this

Reply:
sure thing, will create an issue for this